docs/administration/data/data-migration.md

```diff
@@ -29,6 +29,14 @@ The data migration tools are not suitable for every imaginable scenario. In thes
 1. To move your entire Octopus Server from a self-hosted installation to Octopus Cloud, see [migrating from self-hosted to Octopus Cloud](/docs/octopus-cloud/migrations.md).
 1. To move your entire Octopus Cloud instance to a self-hosted Octopus Server please [contact our support team](mailto:support@octopus.com) for advice and assistance.
 
+## Unsupported scenarios
+
+Sometimes using the data migration tool may look like it could solve a problem, but in fact will make things worse. Here are some scenarios we've seen that are explicitly not supported.
+
+### Export -> Modify -> Import
```

Comment:
An absolute nitpick on formatting that is totally up to you -> I might chuck at least one more hashtag on there to give a nice clear visual difference that we're reading inside a subsection. (.e.g. since right now we've got the one example, it kinda looks like `## tips` is part of the unsupported scenarios).

Nice explicit message! I can see how this would be tempting for users to do, so the warning is a good idea! Does hand manipulating data void warranty or anything severe like that? If so, might be good to mention. 🤷

Reply:
Note on the heading level, docsync will fail validation if the heading levels skip a level, i.e., going from ## (h2) to #### (h4), but I agree it would be good to make this stand out more, perhaps with :::warning which will put the section inside a red callout. I'll modify and commit.